tox.ini

```diff
@@ -67,3 +67,10 @@ commands =
 
 [pytest]
 junit_family = xunit2
+
+[testenv:docs]
+extras=docs
+commands =
+  make -C docs html
+whitelist_externals =
+  make
```

Comment:
Oh, don't use make please, use instead https://github.com/pypa/virtualenv/blob/main/tox.ini#L101-L109

Reply:
This makefile is auto-generated by `sphinx`. E.g.: don't consider it something that you'd maintain, rather an artifact that comes as-is.

Honestly: I'm a bit inclined to prefer an upstream-maintained `Makefile` than those three rather-non-trivial lines of "python". I do think there is strong value in trying to upstream work so that something like `python -m sphinx ...` can do what we want.

That said, if this is a dealbreaker I can go ahead a refactor this bit.

Reply:
It's a dealbreaker. I don't want to have generated and platform-specific code. The documentation should be able to be generated the same way on Windows (without cygwin/wsl/etc) as on Linux.